### DISCUSSION TITLE
Plex album covers

### DISCUSSION THREAD

**Discussion by BeerNecessities** [jac0b-w/album-art-wallpaper#37] [Ideas]
I don't know if adding plex album covers would be easy but it would be nice since I use plex as my media server.  Thanks again!

**Comment by jac0-b** [comment]
> I don't know if adding plex album covers would be easy but it would be nice since I use plex as my media server. Thanks again!

Looks like plex can connect to last.fm based on [this reddit thread.](https://www.reddit.com/r/PleX/comments/gunwii/comment/jfap3ng/?utm_source=share&utm_medium=web2x&context=3) So just create a last.fm account and connect it to that :)

**Reply by BeerNecessities** [reply]
Ah yest it does!  I'll have to switch it manually to last.fm in your app.  It would be nice if it would automatically switch between the two when spotify isn't being used.  Like I posted in another thread here.  Thanks!
